classloader-leak-prevention/src/main/java/se/jiderhamn/classloader/leak/prevention/cleanup/MultiThreadedHttpConnectionManagerCleanUp.java

```diff
@@ -0,0 +1,26 @@
+package se.jiderhamn.classloader.leak.prevention.cleanup;
+
+import se.jiderhamn.classloader.leak.prevention.ClassLoaderLeakPreventor;
+import se.jiderhamn.classloader.leak.prevention.ClassLoaderPreMortemCleanUp;
+
+/**
+ * Invokes static method org.apache.commons.httpclient.MultiThreadedHttpConnectionManager.shutdownAll() to close connections left out by com.sun.jersey.client.apache.ApacheHttpClient.  
+ * 
+ * @author Marian Petrik
+ */
+public class MultiThreadedHttpConnectionManagerCleanUp implements ClassLoaderPreMortemCleanUp {
+	
+  @Override
+  public void cleanUp(ClassLoaderLeakPreventor preventor) {
+	    final Class<?> connManager = preventor.findClass("org.apache.commons.httpclient.MultiThreadedHttpConnectionManager");
+	    if(connManager != null) {
```

Comment:
I found the bug in an application running on JBoss EAP with Jersey dependencies provided by the application server. I tried overriding the dependencies to compile scope newest version, didn't help. I am not going to do anything more about this pull request. Thanks.